en/renew-tls-certificate.md

```diff
@@ -0,0 +1,221 @@
+    kubectl create secret generic ${cluster_name}-tikv-cluster-secret --namespace=${namespace} --from-file=tls.crt=tikv-server.pem --from-file=tls.key=tikv-server-key.pem --from-file=ca.crt=ca.new.pem --dry-run=client -o yaml | kubectl apply -f -
+    kubectl create secret generic ${cluster_name}-tidb-cluster-secret --namespace=${namespace} --from-file=tls.crt=tidb-server.pem --from-file=tls.key=tidb-server-key.pem --from-file=ca.crt=ca.new.pem --dry-run=client -o yaml | kubectl apply -f -
+    kubectl create secret generic ${cluster_name}-cluster-client-secret --namespace=${namespace} --from-file=tls.crt=client.pem --from-file=tls.key=client-key.pem --from-file=ca.crt=ca.new.pem --dry-run=client -o yaml | kubectl apply -f -
+    ```
+
+    In the above command, `${cluster_name}` is the name of the cluster, and `${namespace}` is the namespace deployed for the TiDB cluster.
+
+    > **Note:**
+    >
+    > In the above command, assume that you refer to steps in [renew and replace the TLS Certificate](#renew-and-replace-the-tls-certificate) to back up the new CA certificate as `ca.new.pem`.
+
+2. Perform the rolling restart to components that need to load the combined CA certificate, refer to [perform the rolling restart to the TiDB cluster](restart-a-tidb-cluster.md).
+
+    After the completion of the rolling restart, each component can only accept the certificate issued by the new CA certificate.
+
+## Renew and replace the certificate issued by the `cert-manager`
+
+If the original TLS certificate is issued by [the `cert-manager` system](enable-tls-between-components.md#using-cert-manager), and the original certificate has not expired, you can handle with the issue separately according to whether to renew the CA certificate.
+
+### Renew and replace the CA certificate and certificates between components
+
+When using `cert-manager` to issue the certificate, the `cert-manager` can automatically update the certificate before it expires by specifying the `spec.renewBefore` of the `Certificate` resource.
+
+Although `cert-manager` can automatically renew the CA certificate and the corresponding Kubernetes Secret objects, it currently does not support merging the old and new CA certificates into a combined CA certificate to accept certificates issued by the new and old CA certificates at the same time. Therefore, when renewing and replacing the CA certificate, the issue that TLS between cluster components can not authenticate each other might occur.
+
+> **Warning:**
+>
+> Because the certificates issued by the new and old CAs can not be accepted between components at the same time, renewing and replacing certificates needs to rebuild the Pods of some components. This might cause some requests to access the TiDB cluster to fail.
```

Comment:
```suggestion
> Because the components cannot accept certificates issued by the new and old CAs at the same time, during the process of renewing and replacing certificates, some components' Pods need to be recreated. This might cause some requests to access the TiDB cluster to fail.
```